src/unlimited.cpp

```diff
@@ -2139,10 +2136,6 @@ extern UniValue getstructuresizes(const UniValue &params, bool fHelp)
     ret.pushKV("txInQ", (uint64_t)txInQ.size());
     ret.pushKV("txDeferQ", (uint64_t)txDeferQ.size());
 
-#ifdef DEBUG_LOCKORDER
-    ret.pushKV("lockorders", (uint64_t)lockorders.size());
-#endif
-
```

Comment:
Looks like this got removed in your first commit when you needed to delete the old lock tracking system.  Would be good to put something back in so we can track sizes.

Reply:
I can add the size of all 5 std::maps that are being used now and display that value if you agree that would be roughly the equivalent?

Reply:
i have added this